api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -470,6 +474,28 @@ protected void handleLocationReference(Criteria criteria, ReferenceParam locatio
 		}
 	}
 	
+	protected Optional<Criterion> handleParentReference(ReferenceParam parentReference) {
```

Comment:
Why is this here instead of in the `LocationDaoImpl`? Unlike other reference handlers, this is pretty specific to `Location`s.